tests/unit/test_docs.py

```diff
@@ -0,0 +1,35 @@
+"""Tests for the documentation."""
```

Comment:
cheers Manu! Here's a general point about this test module: I am happy to see a start of actually testing the documentation outside the CI test that just builds it, but I'd be happier to see a dedicated test dir created where we put different test modules for the documentation so e.g. for this case `tests/unit/documentation/test_changelog.py` - what do you think?

Reply:
Sure, can do that, but note that's the exact location the [corresponding ESMValTool test](https://github.com/ESMValGroup/ESMValTool/blob/main/tests/unit/test_docs.py) lives.